metagamer/environments/tictactoe.py

```diff
@@ -17,6 +17,28 @@
 logger = logging.getLogger(__name__)
 
 
+def policy_page_lines(board: np.array, player: int = 1) -> tuple:
+    """
+    return a possible position that hasn't yet been played.
+    This policy tries simply to fill the board from the top left.
+    Args:
+        board: a 3*3 numpy array whose elements are 0, 1, -1
+        player: either 1 or -1, representing
+    representing empty, player1 and player 2,
+
+    Returns: a tuple representing an unfilled coordinate that's selected.
+
+    """
+    row = 0
+    while row in range(board.shape[0]):
+        col = 0
+        while col in range(board.shape[1]):
+            if board[row, col] == 0:
+                return (row, col)
+            col += 1
+        row += 1
```

Comment:
issue: So this is probably my least favorite way to achieve this in code. numpy makes this doable in two lines, but the pythonic approach would be more general

Two suggestions.
1. Pure python - no need to use while loops and maintain row/col in python
```python
for row in range(board.shape[0]):
    for col in range(board.shape[1]):
        if board[row, col] == 0: 
            return (row, col) 
```
2. Numpy - using https://numpy.org/doc/stable/reference/generated/numpy.argmax.html
```python
index = np.argmax(board==0)
return np.unravel_index(index, board.shape)
```

Reply:
Yep, I think 1 is fine, close to mine.  While has risk involved, as I discovered in a first test attempt. I'll change to that one. np solution looks concise, but less intuitive to read.